include/RAJA/exec-agency/raja_agency.hxx

```diff
@@ -93,35 +96,62 @@ using agency_sequential_exec = agency_base<
   decltype(agency::seq)
 >;
 
-template <typename AGENT, typename WORKER>
-struct agency_taskgraph_base { };
```

Comment:
Rearranged things so that the taskgraph and non-taskgraph execution policies are in different locaitons